php/public/automatic_reload.js

```diff
@@ -0,0 +1,13 @@
+if (document.hasFocus()) {
```

Comment:
My understanding is that this check will be triggered once - when the js is loaded and evaluated for the first time.

If the document does not have the focus then the rest will never be executed.
I think `setTimeout` should always be called and the if condition should live inside the function of `setTimeout`. That way we'd periodically check if the document has focus now and reload if that is the case.

Reply:
if the document does not have focus it will display a reload button that is inserted by default. always setting the timeout did not work for me unfortunatley if I remember correctly.